src/pages/SettingsPage/InitialPage.js

```diff
@@ -0,0 +1,168 @@
+import React from 'react';
+import {
+    TouchableOpacity,
+    View,
+} from 'react-native';
+import PropTypes from 'prop-types';
+import _ from 'underscore';
+import {withOnyx} from 'react-native-onyx';
+import Str from 'expensify-common/lib/str';
+import styles from '../../styles/styles';
+import Text from '../../components/Text';
+import {signOut} from '../../libs/actions/Session';
+import ONYXKEYS from '../../ONYXKEYS';
+import {version} from '../../../package.json';
+import AvatarWithIndicator from '../../components/AvatarWithIndicator';
+import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
+import {redirect, redirectToLastReport} from '../../libs/actions/App';
+import {
+    Gear, Lock, Profile, Wallet,
+} from '../../components/Icon/Expensicons';
+import HeaderGap from '../../components/HeaderGap';
+import MenuItem from '../../components/MenuItem';
+import ROUTES from '../../ROUTES';
+
+const propTypes = {
+    /* Onyx Props */
+    // The personal details of the person who is logged in
+    myPersonalDetails: PropTypes.shape({
+        // Display name of the current user from their personal details
+        displayName: PropTypes.string,
+
+        // Avatar URL of the current user from their personal details
+        avatar: PropTypes.string,
+    }),
+
+    // Information about the network
+    network: PropTypes.shape({
+        // Is the network currently offline or not
+        isOffline: PropTypes.bool,
+    }),
+
+    // The session of the logged in person
+    session: PropTypes.shape({
+        // Email of the logged in person
+        email: PropTypes.string,
+    }),
+};
+
+const defaultProps = {
+    myPersonalDetails: {},
+    network: null,
```

Comment:
existing as well, I think it was defaulted to null to maybe make this logic easier `isActive={network && !network.isOffline}`

Reply:
That check would still work wouldn't it?

Reply:
If `network` was an empty object it would incorrectly return true because both `network` and `!network.isOffline` would both be true. maybe a better way of writing this would be defaulting to an obj and checking if `network.isOffline === false`